pagopa/io-app-design-system
# [IOPLT-67] Migrating list-items

## Short description
Migrating ListItems from io-app

## List of changes proposed in this pull request
- ListItemAction
- ListItemIDP
- ListItemInfo
- ListItemInfoCopy
- ListItemNav
- ListItemNavAlert
- ListItemTransaction
- PressableListItemsBase

## How to test
Some jest tests have been added for each component added. Run "yarn test" to check it.
